ci/package-mariner.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+set -euo pipefail
+
+cd /src
+
+. ./ci/install-build-deps.sh
+
+apt-get update
+apt-get install -y git sudo software-properties-common
+sudo add-apt-repository ppa:longsleep/golang-backports
+apt-get update
+apt -y install make tar wget curl rpm qemu-utils golang-1.13-go genisoimage pigz cpio python-pip python3-distutils
+sudo ln -vs /usr/lib/go-1.13/bin/go /usr/bin/go
+mv /.dockerenv /.dockerenv.old 
+
+mkdir -p packages
+
+git clone https://github.com/microsoft/CBL-Mariner.git
+pushd CBL-Mariner
+git checkout "1.0-stable"
+pushd toolkit
+make package-toolkit REBUILD_TOOLS=y
+popd
+mv out/toolkit-*.tar.gz "/src/CBL-Mariner/toolkit.tar.gz"
+popd
+
+make ARCH="$ARCH" PACKAGE_VERSION="$PACKAGE_VERSION" V=1 dist
+
+MarinerRPMBUILDDIR=/src/Mariner-Build
+MarinerSpecsDir=$MarinerRPMBUILDDIR/SPECS/aziot-identity-service
+MarinerSourceDir=$MarinerSpecsDir/SOURCES
+
+# Extract built toolkit in building direectory
+mkdir -p $MarinerRPMBUILDDIR
+pushd $MarinerRPMBUILDDIR
+mv /src/CBL-Mariner/toolkit.tar.gz toolkit.tar.gz
+tar -xzvf toolkit.tar.gz
+popd
+
+# move tarballed IIS source to building directory
```

Comment:
s/IIS/iot-identity-service/

Reply:
I am not quite sure what you wanted changed here. Is there something at the path s/IIS/iot-identity-service/

Reply:
I'm asking you to replace "IIS" with "iot-identity-service"

Reply:
changed IIS to iot-identity-service